src/docfx/build/xref/XrefMap.cs

```diff
@@ -27,39 +27,79 @@ public IEnumerable<XrefSpec> InternalReferences
                 {
                     if (TryGetValidXrefSpecs(uid, specsWithSameUid, out var validInternalSpecs))
                     {
-                        loadedInternalSpecs.Add(GetLatestInternalXrefMap(validInternalSpecs).spec);
+                        var (internalSpec, referencedFile) = GetLatestInternalXrefMap(validInternalSpecs);
```

Comment:
var (uid, (specs, file))

Reply:
The second element is another list of tuples, can not just load it like a tuple